core/src/test/java/de/unistuttgart/iaas/amyassist/amy/core/speech/SpeechRecognitionTest.java

```diff
@@ -0,0 +1,234 @@
+/*
+ * This source file is part of the Amy open source project.
+ * For more information see github.com/AmyAssist
+ * 
+ * Copyright (c) 2018 the Amy project authors.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information see notice.md
+ */
+
+package de.unistuttgart.iaas.amyassist.amy.core.speech;
+
+import java.io.File;
+import java.util.concurrent.CompletableFuture;
+
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
+import static uk.org.lidalia.slf4jtest.LoggingEvent.*;
+
+import javax.sound.sampled.AudioInputStream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import uk.org.lidalia.slf4jtest.TestLogger;
+import uk.org.lidalia.slf4jtest.TestLoggerFactory;
+
+/**
+ * TODO: Description
+ * @author
+ */
+/**
+ * Test Class for the Speech Recognition Systems extending the to be tested Class to gain acces to protected Variabels
+ * 
+ * @author Kai Menzel
+ */
+public class SpeechRecognitionTest extends SpeechRecognizer {
```

Comment:
don't extends the class that should be tested

Reply:
Aber ohne kann ich halt wirklich nichts dieser klasse testen. Ich brauch zugriff auf die zu testenden Methoden.